sw/host/opentitanlib/src/backend/mod.rs

```diff
@@ -47,8 +47,8 @@ pub struct BackendOpts {
     #[structopt(flatten)]
     ti50emulator_opts: ti50emulator::Ti50EmulatorOpts,
 
-    #[structopt(long, help = "Configuration files")]
-    conf: Option<PathBuf>,
+    #[structopt(long, number_of_values(1), help = "Configuration files")]
```

Comment:
Do we also want `multiple = true` here? It appears in `structopt`'s docs but doesn't seem to be in the latest version of `clap`'s docs

Reply:
It seems that with the type being Vec<...>, that structopt does accept multiple instances of the `--conf` switch, I am not sure that adding `multiple = true` would add anything.

Reply:
SG, I thought it didn't accept multiple --conf args